src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -351,12 +354,20 @@ def test_stream_run_logs(self, pf):
     def test_stream_failed_run_logs(self, pf, capfd: pytest.CaptureFixture):
         # (default) raise_on_error=True
         with pytest.raises(InvalidRunStatusError):
-            pf.stream(run="3dfd077a-f071-443e-9c4e-d41531710950")
+            pf.stream(run=FAILED_RUN_NAME_EASTUS)
         # raise_on_error=False
-        pf.stream(run="3dfd077a-f071-443e-9c4e-d41531710950", raise_on_error=False)
+        pf.stream(run=FAILED_RUN_NAME_EASTUS, raise_on_error=False)
         out, _ = capfd.readouterr()
         assert "Input 'question' in line 0 is not provided for flow 'Simple_mock_answer'." in out
```

Comment:
Assert "additionalInfo" exist in this case?

Reply:
I think we don't need to? in my new test, the assert looks like:

<img width="638" alt="image" src="https://github.com/microsoft/promptflow/assets/38847871/bf4d8531-b5bd-46dc-ba0d-4ff90ad32c85">

so it should cover this case